testing/page-loading-spec.js

```diff
@@ -0,0 +1,28 @@
+// Copyright 2017 The TIE Authors. All Rights Reserved.
```

Comment:
I would say allPages.js; anything in tests/ is an e2e test. I would leave *Spec.js for the karma unit tests.